### ISSUE TITLE
multi source to one destination

### ISSUE THREAD

**Issue by Nu-est** [mganss/SyncChanges#179]
@mganss Thanks for your contribution.
is there any multi source feature like that
![image](https://user-images.githubusercontent.com/40880188/229583467-ec992e71-3629-4809-bb9d-28d0fc237f10.png)

**Comment by mganss** [comment]
No, unfortunately there is currently no such feature and there are no plans to add it.
